apps/preview/next/pages/showcases/Pagination/Pagination.tsx

```diff
@@ -105,7 +105,7 @@ export function Showcase() {
                 <div className="flex pt-1 border-t-4 border-transparent">
                   <button
                     type="button"
-                    className="px-4 py-3 rounded-md text-neutral-500 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900 "
+                    className="px-4 py-3 rounded-md text-neutral-500 w-auto md:w-12 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900 "
```

Comment:
```suggestion
                    className="px-4 py-3 rounded-md text-neutral-500 w-auto md:w-12 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900"
```
I'm not sure but do we need this `w-auto` at all? I think its default for `button` so we need only `md:w-12`